web/src/components/ViewOrderModal.jsx

```diff
@@ -34,6 +38,12 @@ export default chakra(function ViewOrderModal({
   isOpen,
   onClose,
 }) {
+  const { isOpen: isConfirmationOpen, onOpen: onConfirmationOpen, onClose: onConfirmationClose } = useDisclosure();
+  const dispatch = useDispatch();
+  const deleteOrder = () => {
+    dispatch(removeOrderAsync(data.orderId));
+    onConfirmationClose();
```

Comment:
This also should close the expanded modal for viewing order if delete is processed. Currently it opens modal for next group order

Reply:
@StevenZhao318 for some reason this doesn't happen to me, but I added an onClose() call to the ViewOrderModal on the next line, does that fix it?